gst-libs/mfx/gstmfxsurface.c

```diff
@@ -228,6 +228,24 @@ gst_mfx_surface_create (GstMfxSurface * surface, const GstVideoInfo * info,
   return TRUE;
 }
 
+static gboolean
+gst_mfx_surface_create_from_buffer_proxy (GstMfxSurface * surface, const GstVideoInfo * info,
```

Comment:
Misleading, no references to VA prime buffer here.

Reply:
The pull request will make more sense for you now.

Reply:
I don't see any changes yet made.

Reply:
We did use this code. 
gst_mfx_surface_vaapi_new_external 
-> gst_mfx_surface_new_external 
    -> gst_mfx_surface_create_from_buffer_proxy

Reply:
OK. For now, it's fine, as long as it doesn't break anything :)